comp230_01_1607539.tex

```diff
@@ -0,0 +1,65 @@
+
+Mods have been considered to be ``hacker art'', an artistic endeavour, and a creative outlet \cite{sotamaa2010game}. Although, one of the oldest fears about art is that it may corrupt observers and lead them to immorality \cite{schulzke2009moral}. In games, the amount of violence usually depends heavily on how a player chooses to act \cite{schulzke2010defending}. Ethically notable games either attempt to make the player feel responsible for the decisions they make in the game, encode an ethical system and require the player to follow it in order to succeed, or provide players with situations in which their understanding of the ethical system is challenged \cite{zagal2009ethically}. Then can a game that only allows a good course of action be ethically correct if it does not allow free will? Considering the opposite, is Danny Ledonne's `linear' game \textit{Super Columbine Massacre RPG!} condemned simply because it is a game about a serious and emotional topic? Perhaps the act of playing a game trivializes the issues it tackles and thus renders any game about a serious topic inherently unethical \cite{zagal2009ethically}. Game developers should consider how the player can behave ethically, more than the social impact of the narrative.
+
+We should recognize that in the modding community there is resistance to governance. That there are those who choose to ignore the license terms and conditions that come with closed, restricted games \cite{scacchi2010computer}. Even a strict modding policy that outlines exactly what is and is not permissible, such as Rockstar has with \textit{GTA V}, can result in controversial cease and desist orders \cite{gta1}. Nintendo are also extremely protective of their intellectual property, regularly serving takedown notices to modders who dilute the \textit{Mario}, \textit{Zelda} \cite{nintendo1}, or \textit{Pok{\'e}mon} \cite{pokemon1} brands. Value is derived from sales of software, and goodwill can accrue with an application's reputation for technical integrity \cite{herman2006your}. However, this can backfire against the developer. For example, copyright disputes between corporations and consumers over file sharing and sampling in the music industry have tarnished the goodwill of consumers towards certain bands (i.e. \textit{Metallica}) perceived to be overzealous in enforcing their intellectual property rights \cite{herman2006your}. 
```

Comment:
The term `goodwill` has a particular meaning in legal discussion. Consider using a synonym if that is not what you mean.